build.yaml

```diff
@@ -318,6 +318,7 @@ common_cli:
 
     # using the caching-layer, install all the dependencies
     pip install --upgrade pip
+    pip install -r requirements.txt ${PIP_ADDITIONAL_OPTIONS}
     pip install . ${PIP_ADDITIONAL_OPTIONS}
     pip install ".[dev]" ${PIP_ADDITIONAL_OPTIONS}
```

Comment:
:question: Unrelated to this branch but why do we install dev requirements in separate step? 

And I was going to add "And in my experience this command doesn't work anyway. It has to be `pip install Flocker[dev]`."

But it seem newer `pip`s handle it.